undeploy.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
This whole script is now nothing more than a wrapper around `cdk destroy`. Most of what's still in here is now superfluous and should be deleted.

Reply:
An even simpler approach would be to replace it with a `bash` script that simply executes `cdk destroy`.

Reply:
If you want to keep the python wrapper, that makes sense, but delete all the unused parts of this file (the unused imports, commented out code, etc). Wouldn't hurt to run `flake8` over it also.